cli/flox/src/commands/mod.rs

```diff
@@ -243,12 +244,35 @@ impl FloxArgs {
 #[allow(clippy::large_enum_variant)] // there's only a single instance of this enum
 #[derive(Bpaf, Clone)]
 enum Commands {
+    /// Prints help information
+    #[bpaf(command, hide)]
+    Help(#[bpaf(external(help))] Help),
     Development(#[bpaf(external(local_development_commands))] LocalDevelopmentCommands),
     Sharing(#[bpaf(external(sharing_commands))] SharingCommands),
     Additional(#[bpaf(external(additional_commands))] AdditionalCommands),
     Internal(#[bpaf(external(internal_commands))] InternalCommands),
 }
 
+#[derive(Debug, Bpaf, Clone)]
+struct Help {
+    /// Command to show help for
+    #[bpaf(positional("cmd"))]
+    cmd: Option<String>,
+}
+impl Help {
+    fn handle(self) {
+        let mut args = Vec::from_iter(self.cmd.as_deref());
+        args.push("--help");
+
+        match flox_cli().run_inner(&*args) {
+            Ok(_) => unreachable!(),
+            Err(ParseFailure::Completion(comp)) => print!("{comp}"),
+            Err(ParseFailure::Stdout(doc, _)) => info!("{doc}"),
+            Err(ParseFailure::Stderr(err)) => error!("{err}"),
```

Comment:
nonblocking: I don't really understand what the Completion or Stderr arms are doing - are those reachable?

Reply:
I confirmed stderr might be at some point but isn't now
The completions might bereached if you put the right flags behind -- but should be catched earlier

Generally neither should be reachable in normal circumstances